js/views/onboardingModal.js

```diff
@@ -351,6 +351,10 @@ module.exports = baseModal.extend({
                 console.log(errorThrown);
               }
             });
+          } else {
```

Comment:
What is the reason the settings is failing?

Because... it's a little weird to show the connection modal in this case, because if the endpoint is responding with a reason, the server is not down. Furthermore, if the server were really down, the heartbeat socket would close which would cause the connect modal to come up.

What this code in this PR would do is show the connect modal with a UI state reflecting that the server is down, when a connection to the server actually exists. I'm not sure how it would handle that.

If we know the reason that settings is failing, I think we could come up with a better experience. Maybe a Dialog with the reason and 'Try again' & 'Restart' buttons...?